### DISCUSSION TITLE
lots of pod bad request problems

### DISCUSSION THREAD

**Discussion by ssahinh** [abpframework/abp#18420] [Q&A]
when I run my application on more than 1 pod on Kubernetes, I get a bad request when I reach some endpoints, but when I reduce the number of pods to 1, the problem goes away. why could it be?

**Comment by maliming** [comment]
Please check your logs first.

**Comment by ssahinh** [comment]
there is no error visible in the logs. that's why I opened a discussion here.

**Reply by maliming** [reply]
> I get a bad request 

You can share the logs of bad request.

**Reply by maliming** [reply]
Please share the full logs.

`Logs.txt`
